tests/conftest.py

```diff
@@ -46,26 +47,39 @@ def load_meta(pdir, meta_file):
 
 for ddir in os.listdir(DATA_DIR):
     pdir = DATA_DIR / ddir
-    meta_file = pdir / "meta.yaml"
-    if not pdir.is_dir():
-        continue
-    if not meta_file.is_file():
-        continue
-
-    metadata = load_meta(pdir, meta_file)
-    VTK_FILES.update({metadata["id"]: metadata["attrs"]})
+    for filename in os.listdir(pdir):
+        if filename.startswith("meta") and filename.endswith(".yaml"):
+            meta_file = pdir / filename
+            if not pdir.is_dir():
+                continue
+            if not meta_file.is_file():
+                continue
+            metadata = load_meta(pdir, meta_file)
+            if "XDMF" in metadata["attrs"]["kind"]:
+                XDMF_FILES.update({metadata["id"]: metadata["attrs"]})
+            else:
+                VTK_FILES.update({metadata["id"]: metadata["attrs"]})
```

Comment:
I did this so that multiple `meta` files describing different files in the same directory can be read in for different tests. See for example, `tests/data/pluto_isentropic_vortex`

Reply:
Ah, I missed that. Ok then, thanks for the clarification

Reply:
It is this one: [c667985](https://github.com/dutta-alankar/yt_idefix/commit/c6679a50b81badd997f93a877f2ce6aee6dcf909)
![image](https://user-images.githubusercontent.com/39578361/222036157-164508a9-1a76-4e73-bd2f-542e9e3e447a.png)
Sorry for the late reply. It was night at my time zone and I was sleeping.



> I haven't had time today to remind myself of how adding large files to this repo works (this is my only project using git-lfs), but I am now having issues checking in and out of this branch locally, seemingly because the couple files you added were committed as ordinary files and shouldn't have. I will probably need to rewrite the history of this branch sooner than I expected to fix the problem. In the mean time, can you pin point the commit(s) that added `tests/data/pluto_isentropic_vortex/data.0010.vtk` ?